src/Compilers/CSharp/Portable/Binder/Binder_Operators.cs

```diff
@@ -435,6 +435,18 @@ private BoundExpression BindSimpleBinaryOperator(BinaryExpressionSyntax node, Di
 
             if (left.HasAnyErrors || right.HasAnyErrors)
             {
+                if (node.OperatorToken.IsKind(SyntaxKind.MinusToken)
+                    && !right.HasAnyErrors
+                    && left.Kind == BoundKind.BadExpression
+                    && left.Type != null
+                    && left.Type.Kind == SymbolKind.NamedType
+                    && node.Left != null
+                    && node.Left.IsKind(SyntaxKind.ParenthesizedExpression)
+                    && ((node.Left as ParenthesizedExpressionSyntax)?.Expression?.IsKind(SyntaxKind.SimpleMemberAccessExpression) ?? false))
```

Comment:
>Expression? [](start = 70, length = 11)

Can expression ever be null? Please get rid of the conditional access otherwise.